Documentation/kubernetes-on-aws-render.md

```diff
@@ -0,0 +1,255 @@
+# Configure your Kubernetes cluster on AWS
+
+This is the second step of [running Kubernetes on AWS][aws-step-1]. Before we launch our cluster, let's define a few parameters that the cluster requires.
+
+## Cluster parameters
+
+### EC2 key pair
+
+The keypair that will authenticate SSH access to your EC2 instances. A single key will be configured on each CoreOS node for debugging any issues you may encounter.
+
+After creating a key pair, you will use the name you gave the keys to configure the cluster. Key pairs are only available to EC2 instances in the same region. More info in the [EC2 Keypair docs](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html).
+
+### KMS key
+
+[Amazon KMS](http://docs.aws.amazon.com/kms/latest/developerguide/overview.html) keys are used to encrypt and decrypt cluster TLS assets. If you already have a KMS Key that you would like to use, you can skip creating a new key and provide the Arn string for your existing key.
+
+Creating a KMS key can be done via the [AWS web console](http://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html) or via the AWS cli tool:
```

Comment:
You can create a KMS key in the [AWS console](http://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html) or with the `aws` command line tool: